src/DynamicRupture/FrictionLaws/LinearSlipWeakening.cpp

```diff
@@ -10,27 +17,19 @@ void BiMaterialFault::copyLtsTreeToLocal(seissol::initializers::Layer& layerData
   regularisedStrength = layerData.var(concreteLts->regularisedStrength);
 }
 
-void BiMaterialFault::strengthHook(real& strength,
+real BiMaterialFault::strengthHook(real& faultStrength,
                                    real& localSlipRate,
-                                   real& sigma,
-                                   real& mu,
                                    real& deltaT,
                                    unsigned int ltsFace,
                                    unsigned int pointIndex) {
   // modify strength according to Prakash-Clifton
   // see e.g.: Pelties - Verification of an ADER-DG method for complex dynamic rupture problems
-  prak_clif_mod(strength, sigma, localSlipRate, mu, deltaT);
-  // save for output
-  regularisedStrength[ltsFace][pointIndex] = strength;
+  real expterm = std::exp(-(std::abs(localSlipRate) + drParameters.vStar) * deltaT /
```

Comment:
localSlipRate should not be negative. But maybe it is because of the resampling?
But in case it gets negative I would rather use max(0, localSlipRate) in the formula.

Reply:
ok, then let's keep it as it is.